app/assets/javascripts/upload_setup.js

```diff
@@ -0,0 +1,28 @@
+/*eslint no-unused-vars: ["error", { "vars": "local" }]*/
+/*global uploadSetup */
```

Comment:
dumb question: don't we have to include this in the manifest or a view file in order for it to be available?